sites/svelte.dev/package.json

```
@@ -54,7 +55,7 @@
     "svelte-preprocess": "^5.0.4",
     "tiny-glob": "^0.2.9",
     "typescript": "^5.1.6",
-    "vite": "^4.3.9",
+    "vite": "^4.4.0",
```

Vite 4.4.2 is out now
```suggestion
    "vite": "^4.4.2",
```